modules/async-icq/testing/simapp/upgrades/upgrades.go

```diff
@@ -0,0 +1,68 @@
+package upgrades
+
+import (
+	"github.com/cosmos/ibc-apps/modules/async-icq/v7/keeper"
+	icqtypes "github.com/cosmos/ibc-apps/modules/async-icq/v7/types"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	consensusparamskeeper "github.com/cosmos/cosmos-sdk/x/consensus/keeper"
+	paramskeeper "github.com/cosmos/cosmos-sdk/x/params/keeper"
+	paramstypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+)
+
+const (
+	V2 = "v2"
+)
+
+// CreateDefaultUpgradeHandler creates an upgrade handler which can be used for regular upgrade tests
+// that do not require special logic
+func CreateDefaultUpgradeHandler(
+	mm *module.Manager,
+	configurator module.Configurator,
+) upgradetypes.UpgradeHandler {
+	return func(ctx sdk.Context, _ upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+		return mm.RunMigrations(ctx, configurator, vm)
+	}
+}
+
+// We will have to import every one here
+func CreateV2UpgradeHandler(
```

Comment:
nit: godoc comments should begin with the function/method name and use complete sentences with appropriate punctuation.